cmd/cli/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package main in cli is the command line interface for the application.
```

Comment:
`in cli` no need to be such specific, if you move this under another folder, you will need to change this.
this info doesn't help to understand this pkg more.